scripts/load_mentors.js

```diff
@@ -133,6 +139,17 @@ exports.handler = async (event) => {
 
   if (deletedMentorIds.length > 0) {
     console.log("Deleting old mentors...");
+    [...deletedMentorIds].flatMap((mentorId) => {
+      [`${mentorId}-full`, `${mentorId}-thumbnail`].map((imageKey) => {
+        s3.deleteObject(
+          {
+            Bucket: AWS_S3_BUCKET_NAME,
+            Key: `${AWS_S3_IMAGE_FOLDER}${imageKey}.jpg`,
+          },
```

Comment:
Could we abstract this out into an arrow function taking in `imageKey`? I see that this was also used in L177-183.